config/locales/views/en.yml

```diff
@@ -372,9 +378,62 @@ en:
           intro: "Unblocking a user allows them to resume:"
           commenting: "commenting on your works"
           replying: "replying to your comments anywhere on the site"
+  muted:
+    mute: "Mute"
+    unmute: "Unmute"
+    muted_items_notice: "You have muted some users on the Archive. Some items may not be shown, and any counts may be inaccurate. You can mute or unmute users on %{muted_users_link}."
+    muted_users_link_text: "your Muted Users page"
+    users:
+      index:
+        title: "Muted Users"
+        button: "Mute"
+        none: "You have not muted any users."
+        legend: "Mute a user"
+        label: "User"
+        heading:
+          landmark:
+            muted_users: "Listing Muted Users"
+        will:
+          intro: "Muting a user:"
```

Comment:
PAC and Support would like one bit of text added here (just on the index, not the confirmation page). They'd like it to mention the limit and say: "You can mute up to [limit, with delimiter] users. Muting a user:"